### DISCUSSION TITLE
Adding video paper walkthroughs as additional resources for each model

### DISCUSSION THREAD

**Discussion by mdanb** [pyg-team/pytorch_geometric#2496] [Ideas]
I think it would be helpful for the community to have video walkthroughs for the research papers associated with each of the different graph models, similar to what is done [here](https://m.youtube.com/watch?v=54uLU7Nxyv8) for different NLP models. Perhaps we can have a link to the videos below each model’s description. We could also maybe setup a slack channel so that people who are making the videos and have questions about them can get help from the community. It could act as a space for people to ask questions about the videos / papers. 
I’m might, depending on my time commitments, be able to start this initiative at some point if this sounds like something that might be of interest at all. Just want to get an idea of whether this would be helpful first.

**Comment by rusty1s** [comment]
Definitely agree :) We have plans to extend the tutorials and examples soon. Integrating videos is definitely a good option. If you are interested in doing that, I will definitely link to it in the PyG documentation and examples section :)

There's also the [PyTorch Geometric Tutorial](https://antoniolonga.github.io/Pytorch_geometric_tutorials/) initiative that is similar to what you are proposing :)

**Reply by adhikarirsr** [reply]
First of all, thanks a lot @rusty1s for all your hard work. 

This sounds like a good idea. Even to begin with just adding description at the beginning of the codes in the example section and commenting pertinent parts of codes would be super useful or  may be creating well-described google colab version of the example codes would be extermely helpful. 

I really enjoyed and learnt a lot from the google colab tutorial. It would be really helpful to have similar tutorial on how to convert one of the GNN's paper into a fully functioning GNN layer. This might help to accelerate the development of PyG ecosystem.
